src/controllers/profile.controller.js

```diff
@@ -0,0 +1,131 @@
+import { ApiError } from '../exceptions/api.error.js';
+import { userService } from '../services/user.service.js';
+import { User } from '../models/user.js';
+import bcrypt from 'bcrypt';
+import { validatePassword } from '../utils/methods.js';
+import { emailService } from '../services/email.services.js';
+import jwt from 'jsonwebtoken';
+
+const rename = async (req, res) => {
+  const { newName, email } = req.body;
+
+  if (!newName) {
+    throw ApiError.badRequest('Bad request, need to provide new name of user');
```

Comment:
The error message 'Bad request, need to provide new name of user' could be more consistent with other error messages by capitalizing 'user' to 'User'.